src/template.test.ts

```diff
@@ -89,5 +89,37 @@ describe("templating", () => {
         },
       ]);
     });
+
+    it("Should format a chat template with a chat history role", () => {
```

Comment:
this test is a copy of one right now, I'm working on a real test for chat_history